package.json

```diff
@@ -107,6 +107,8 @@
     "start": "RADICLE_UPSTREAM_EXPERIMENTAL=false yarn start:app",
     "start:experimental": "RADICLE_UPSTREAM_EXPERIMENTAL=true yarn start:app",
     "start:app": "run-p --race rollup:watch proxy:start electron:start",
+    "start:release": "run-p --race rollup:watch proxy:start:release electron:start",
```

Comment:
:raised_hands: 

Although, I think it should be the other way around. `start` should be release and then `start:debug` is the debug build, or whatever you want to call it.

Reply:
Is this so that the unlocking and pushing works faster? Do we lose anything by not having proxy in debug mode? If so we could run it by default as @FintanH suggests.

Reply:
We lose time every time we wait for a git operation :upside_down_face: